src/topology/algebra/affine.lean

```diff
@@ -25,7 +27,8 @@ section ring
 
 variables [ring R] [module R E] [module R F]
 
-/-- An affine map is continuous iff its underlying linear map is continuous. -/
+/-- An affine map is continuous iff its underlying linear map is continuous. See also
+`affine_map.continuous_linear_iff`. -/
 lemma continuous_iff {f : E →ᵃ[R] F} :
```

Comment:
@jcommelin raised questions about naming which I have marked as resolved but this old lemma might deserve a rename and also to be restated as `continuous f.linear ↔ continuous f` for consistency with `affine_map.continuous_linear_iff`.

Since I don't actually use this old lemma (I'm just updating its doc string) and I have two other PRs dependent upon this one, I'd vote for carrying out such tidy up as a follow-up PR but of course I'm happy to do whatever others prefer.